### PR TITLE
updating deployment readme and docs/readme

### PR DESCRIPTION
**Pull request by chinmayic**
**Deployment Readme changes:**
1.  Changed odim to odimra in:
     $ docker exec -it odim_1 /bin/bash
    Edit the parameters in the odim_config.json file located in this path: /etc/odim_config/odim_config.json and save.

2. Added Docker proxy content.
3. Semantic and formatting changes.

**Docs/readme changes:**

1. Semantic and formatting changes.
2. Fixed broken internal reference links.
3. Added "Required privileges" column for the list of APIs under every section.
4. Moved content under "User roles and privileges" to a separate section called  "role-based authorization"  under "Authentication and authorization".

### CODE REVIEW

**Review by rahulroshan-kachchap — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -300,31 +311,39 @@ This section provides a step-by-step procedure for deploying the resource aggreg
...
   
-  #  Modifying default configuration parameters for the resource aggregator
+#  Modifying default configuration parameters for the resource aggregator
 
-1.   Navigate to the `odim_1` container using the following command: 
+1.   Navigate to the `odimra_1` container using the following command: 
 
       ```
-     $ docker exec -it odim_1 /bin/bash
+     $ docker exec -it odimra_1/bin/bash
```
Comment (nimmalagautam) [inline README.md:359]: docker exec -it odimra_1/bin/bash

the command is wrong, Please change it

correct command:  docker exec -it build_odimra_1 /bin/bash

File: README.md [hunk 2]
```diff
@@ -300,31 +311,39 @@ This section provides a step-by-step procedure for deploying the resource aggreg
...
+password: GRFPlug!n12$4
+``` 
+ 
+ 
   
-  #  Modifying default configuration parameters for the resource aggregator
+#  Modifying default configuration parameters for the resource aggregator
 
-1.   Navigate to the `odim_1` container using the following command: 
+1.   Navigate to the `odimra_1` container using the following command:
```
Comment (nimmalagautam) [inline README.md:356]: Navigate to the `build_odimra_1` container using the following command:

File: README.md [hunk 3]
```diff
@@ -369,8 +388,82 @@ This section provides a step-by-step procedure for deploying the resource aggreg
 
      ```
      $ docker restart odim_1
```
Comment (nimmalagautam) [inline README.md:406]: docker restart build_odimra_1

File: README.md [hunk 4]
```diff
@@ -369,8 +388,82 @@ This section provides a step-by-step procedure for deploying the resource aggreg
...
+|---------|-----------|
+|`<Proxy_URL>` |Your company URL.|
+|`<ODIM_server_VM_IP>` |The IP address of the system where the resource aggregator is installed.|
+|`<FQDN>` |FQDN of the resource aggregator server.|
+
+</blockquote>
+
+### Procedure
+
+1.   In the home directory of odimra user, create a hidden directory called .docker, and then create a file called config.json.
```
Comment (nimmalagautam) [inline README.md:431]: please provide the command to create the hidden directory and mention that go to the .docker directory and then create a file config.json

$ mkdir .docker
$ cd .docker
$ vi config.json

**Review by nimmalagautam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chinmayic) [reply README.md]: Changed to the right command.

File: README.md (hunk 3 above)
Comment (chinmayic) [reply README.md]: Changed to the right command.

File: README.md (hunk 2 above)
Comment (chinmayic) [reply README.md]: Changed.

File: README.md (hunk 4 above)
Comment (chinmayic) [reply README.md]: Added these commands.

File: README.md (hunk 1 above)
Comment (shivacharanms) [reply README.md]: Space is missing after 1

**Review by nimmalagautam — APPROVED** [review]
The new changes have been updated  for below

creating a proxy for the docker
changing the odim container name to bundle_odimra_1

**Review by bharath-b-hpe — APPROVED** [review]

**Review by nimmalagautam — APPROVED** [review]

**Pull request merged**